packages/wdio-selenium-standalone-service/package.json

```diff
@@ -42,6 +41,8 @@
   },
   "types": "selenium-standalone-service.d.ts",
   "devDependencies": {
-    "@types/fs-extra": "^9.0.1"
+    "@types/fs-extra": "^9.0.1",
+    "webdriver": "^6.6.2",
+    "webdriverio": "^6.6.2"
```

Comment:
```suggestion
    "webdriver": "6.6.2",
    "webdriverio": "6.6.2"
```

Reply:
Also if we only import types from these packages we don't need to declare them as dev dependencies. Given the project setup, TS will find the packages.

Reply:
Yes! Let's remove both webdriver and webdriverio from here, nice catch!